www/common/common-ui-elements.js

```diff
@@ -623,13 +623,19 @@ define([
         var appType = (common.getMetadataMgr().getMetadata().type || 'pad').toUpperCase();
         data = data || {};
         if (!callback && data.callback) { callback = data.callback; }
+
+        function createMenuItemButton(iconClasses, buttonClasses, title, text, ariaLabel) {
+            return $('<button>', {
+                'class': buttonClasses,
+                'title': title,
+                'aria-label': ariaLabel
+            }).append($('<i>', {'class': iconClasses}))
+              .append($('<span>', {'class': 'cp-toolbar-drawer-element'}).text(text));
+        }
```

Comment:
We almost never use named functions in the rest of the product, so we should probably use an anonymous function here too (using `let createMenuItemButton = function (...) {};` syntax).
We also try to avoid using jQuery whenever we can and this is a good example of a place where we can use hyperscript (the `h` function) instead.

About the content, I think we should only append the `<i>` tag if 'iconClasses' is not empty and the `<span>` tag if 'text' is not empty.
And I also think the "ariaLabel" argument is not necessary. Its value should probably always be set to title and fallback to text.